docs/preview/03-Guidance/use-with-dotnet-and-functions.md

```diff
@@ -54,16 +54,13 @@ namespace Arcus.Samples.AzureFunction
 
             builder.Services.AddLogging(loggingBuilder =>
             {
-                loggingBuilder.ClearProvidersExceptFunctionProviders();
```

Comment:
Since we deprecated this method, let's also remove it in our docs.

Reply:
Thought I did that, but must have forgotten it.